Enterprise-CMCS/managed-care-review
# MCR-4594: Change history updates for approvals

## Summary

As a CMS and state user, 
I want to see status change activity in change history table on the submission summary page 
so that I can get a full understanding of all updates made to a submission over time. 

AC: 

1. If a submission has been marked as "approved" 
Then the "Status updated" activity is logged on the change history table with the status, the date of the change, who did the change, and the optional note if any. (See Figma [here|https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=15586-47657&t=yajSHsYdS98zO4Iw-1]; See alternate views of the change history table [here](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=15738-76172&t=yajSHsYdS98zO4Iw-1)

#### Related issues
https://jiraent.cms.gov/browse/MCR-4594

#### Screenshots
<img width="973" alt="Screenshot 2024-11-21 at 2 50 06 PM" src="https://github.com/user-attachments/assets/4c3be82d-beda-4511-b7ed-a3d2b353e786">

Without optional note
<img width="978" alt="Screenshot 2024-11-21 at 2 56 26 PM" src="https://github.com/user-attachments/assets/865e6de7-0534-49bd-b8fe-6454b5d5c804">


#### Test cases covered

- Content of the accordion is correct for an approval action

## QA guidance

Approve a submission and ensure the change history is populated per designs for the approval action
